File: src/main/java/com/udacity/store/model/Order.java
package com.udacity.store.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Table(name = "orders")
@Data
@AllArgsConstructor
@NoArgsConstructor

public class Order {
//TODO: Add the details of the order class
@Id
@GeneratedValue(strategy = GenerationType.IDENTITY)
@Column(updatable = false, nullable = false)
private long id;
    @Column
    private String name;
    @Column
    private double total;
    @Column
    private String creditCart;
    @Column
    private String address;
}

File: src/main/webapp/angular-frontend/src/app/models/Order.ts
//TODO: Add the contents of the order class

export class Order {
    address: string;
    creditCart:string;
    id: number;
    name: string;
    total: number;


    constructor() {
      this.id = 0;
      this.name = '';
      this.total = 0;
      this.address = '';
      this.creditCart = '';
    }
  }
